documentation/sphinx/source/administration.rst

```diff
@@ -488,6 +488,19 @@ If a process has had more than 10 TCP segments retransmitted in the last 5 secon
 
       10.0.4.1:4500       ( 3% cpu;  2% machine; 0.004 Gbps;  0% disk; REXMIT! 2.5 GB / 4.1 GB RAM  )
 
+Machine-readable status
+--------------------------------
+
+The status command can provide a complete summary of statistics about the cluster and the database with the ``json`` argument. Full documentation for ``status json`` output can be found :doc:`here <mr-status>`.
+From the output of ``status json``, operators can find useful health metrics to determine whether or not their cluster is hitting performance limits.
+
+====================== =========================================================== ===================================================
+Ratekeeper limit        ``cluster.qos.transactions_per_second_limit``               The number of read versions per second that the cluster can give out. A low ratekeeper limit indicates that the cluster performance is limited in some way. The reason for a low ratekeeper limit can be found at ``cluster.qos.performance_limited_by``. ``cluster.qos.released_transactions_per_second`` describes the number of read versions given out per second, and can be used to tell how close the ratekeeper is to throttling.
+Storage queue size      ``cluster.qos.worst_queue_bytes_storage_server``            The maximum size in bytes of a storage queue. Each storage server has mutations that have not yet been made durable, stored in the storage queue. If this queue gets too large for any storage server, the ratekeeper will increase throttling. Storage servers with large storage queues have fallen behind and will likely struggle to serve read requests.
+Durable version lag     ``cluster.qos.worst_durability_lag_storage_server``         The ``versions`` field contains the maximum number of versions in a storage queue. Ideally, this should be near 5 million. The ``seconds`` field contains the maximum number of seconds of non-durable data in a storage queue. Ideally, this should be near 5 seconds. If a storage server is overwhelmed, the durability lag could rise, causing performance issues. The sec
+Transaction log queue   ``cluster.qos.worst_queue_bytes_log_server``                The maximum size in bytes of the mutations stored on a transaction log that have not yet been popped by storage servers.
```

Comment:
We might want to call out that if this gets large, ratekeeper will begin throttling. The complicated thing about this, though, is that it is designed to get somewhat large during failures and that shouldn't itself be alarming. It's easiest to express this with numbers, though I'm not certain if we want to include our knob values here. Currently, though, the log queue is expected to grow up to 1.5GB during failures, at which point it starts spilling to disk. If it gets up to 2GB, it will start throttling in increasing amounts to try to limit the queue to 2.4GB.

Reply:
I added a comment for this, but left out numbers and details, since those could easily change in the future.